### PR TITLE
docs: add Pato Vargas to GDE resources

### PR DESCRIPTION
**Pull request by devpato**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [x] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -886,5 +886,13 @@
     "website": "https://alvarocamillont.dev/",
     "bio": "Alvaro is a software engineer, speaker, and instructor in Brazil. He has worked in the technology industry for over 10 years and is dedicated to the development of enterprise business solutions.",
     "groups": ["GDE"]
+  },
+  "devpato": {
+    "name": "Patricio Vargas",
+    "picture": "patovargas.png",
+    "twitter": "devpato",
+    "website": "https://developers.google.com/community/experts/directory/profile/profile-patricio_vargas",
+    "bio": "Pato is Google Developer Expert on Angular & Web Technologies, Auth0 Ambassador, Woment Techmaker Ambassador and Media Developer Expert at Cloudinary. He loves mentoring people in technology, write articles, contribute to open source, speak at conferences/meetups, and advocate technologies that he loves.",
```
Comment (petebacondarwin) [inline aio/content/marketing/contributors.json:895]:
```suggestion
    "bio": "Pato is Google Developer Expert on Angular & Web Technologies, Auth0 Ambassador, Women Techmaker Ambassador and Media Developer Expert at Cloudinary. He loves to mentor people in technology, write articles, contribute to open source, speak at conferences/meetups, and advocate technologies that he loves.",
```

**Review by petebacondarwin — COMMENTED** [review]
A couple of typos but otherwise LGTM.

**Comment by devpato** [comment]
> A couple of typos but otherwise LGTM.

Thank you sir!

**Review by petebacondarwin — APPROVED** [review]
Reviewed-for: global-docs-approvers

**Comment by mhevery** [comment]
merged as 61b9adb1b1

**Pull request closed without merging**